test/integration/wyatt_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class WyattTest < ActionDispatch::IntegrationTest
+  test "shows specific message when visiting '/wyatt'" do
+    get "/wyatt"
+    assert_response :success
```

Comment:
If the test runs and the request results in a HTTP status code other than a 200 the test will fail.